shop/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
unless you are doing something hacky or unconventional I like to just not use comments. They generally  pollute code and get over used. Keep things simple and clean. The less lines of code the better